subsys/usb/class/hid/Kconfig

```diff
@@ -45,4 +45,29 @@ config USB_HID_REPORTS
 	  Number of HID reports in the instance.
 	  Must be equal or higher than highest report ID (if they are not consecutive).
 
+config USB_HID_BOOT_PROTOCOL
+	bool
+	prompt "Enable USB HID Boot Protocol handling"
+	help
+	  Sets bInterfaceSubClass to 1 and enables Set_Protocol and Get_Protocol
+	  requests handling.
+	  See Chapter 4.2 of Device Class Definition for Human Interface 1.11
```

Comment:
Device

Reply:
Looks good to me.

Reply:
Should it be `Human Interface Device` instead of `Human Interface 1.11`

Reply:
You are right, sorry. Fixed.